text/0000-image-labels.md

```diff
@@ -0,0 +1,69 @@
+# Meta
+[meta]: #meta
+- Name: Image Labels
+- Start Date: 2020-03-24
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+Buildpacks should be able to expose arbitrary label metadata on images.  This content should be exposed by enhancing buildpack artifacts.
+
+# Motivation
```

Comment:
can you add a concrete example of why people would use this feature in the buildpack?